electron/constants.ts

```diff
@@ -30,11 +31,18 @@ function getLegendaryBin() {
 
 const isMac = platform() === 'darwin'
 const isWindows = platform() === 'win32'
+const isFlatpak = execPath === '/app/main/heroic'
 const currentGameConfigVersion: GameConfigVersion = 'v0'
 const currentGlobalConfigVersion: GlobalConfigVersion = 'v0'
-const home = homedir()
-const legendaryConfigPath = `${home}/.config/legendary`
-const heroicFolder = `${home}/.config/heroic/`
+const flatPakHome = env.XDG_DATA_HOME.replace('/data', '')
+const home = isFlatpak ? flatPakHome : homedir()
+const legendaryConfigPath = isFlatpak
+  ? `${home}/config/legendary`
+  : `${home}/.config/legendary`
+console.log({ home, legendaryConfigPath })
+const heroicFolder = isFlatpak
+  ? `${home}/config/Heroic/`
+  : `${home}/.config/Heroic/`
```

Comment:
@flavioislima 
You can use XDG_CONFIG_HOME, to get right config path in sandbox  
XDG_CONFIG_HOME -> ~/.var/app/<id>/config

Reply:
Ah, I swear I looked for this one and could not find it. I'll check it.